recipes/stakesign/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "stakesign" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 68fac3b8ee4d8d5ea5f0d70e40df8e70e3b8905ea0bb57d4ba9b38961129e38e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - stakesign = stakesign:main
+  skip: True  # [win]  -- windows unsupported rn
```

Comment:
This recipe looks like it's pure Python--is there an actual need to be able to skip Windows and not have this as `noarch: python`?

Reply:
Good question-- at the moment, the program subprocess-calls the sha256sum utility from coreutils, which is not available on windows AFAICT. The failing build that led me to add the selector is here: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=255817&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77

We'd like to remove the sha256sum dependency in the future, but it's expedient for now to avoid reinventing its various code paths for dealing with potentially malicious inputs.

Reply:
Go ahead and make the recipe `noarch: python`. When this becomes a feedstock, it will only build one package for all platforms (using Linux)--the `coreutils` dependency will keep it from being installed on windows.

And just a general comment (doesn't matter for the recipe), I'm surprised the standard library [`hashlib`](https://docs.python.org/3/library/hashlib.html) module doesn't have all the same code paths that `sha256sum` does.